File: HeartBeatSystemDemo/HeartBeatSystemDemo/src/models/BPMChange.java
package models;

public class BPMChange {
    private int upLimit = HeartBeatSystem.DEAD_LINE_HIGHEST;
    private int downLimit = HeartBeatSystem.DEAD_LINE_LOWEST;
    private int changeValue;
    public BPMChange(int up , int down , int change){
        upLimit = up;
        downLimit = down;
        changeValue = change;
    }
    public int getUpLimit(){
        return  upLimit;
    }
    public int getDownLimit(){
        return downLimit;
    }
    public int getBPMChange(){
        return changeValue;
    }
    @Override
    public String toString(){
        return downLimit + "-" + upLimit + " " + changeValue;
    }
}

File: UnityProjet/scripts/HeartBeatSystem/models/BPMChange.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public struct BPMChange
{
    public readonly int upLimit;
    public readonly int downLimit;
    public readonly int changeValue;
    public BPMChange(int up, int down, int change)
    {
        upLimit = up;
        downLimit = down;
        changeValue = change;
    }
    override
    public readonly string ToString()
    {
        return downLimit + "-" + upLimit + " " + changeValue;
    }
}
